### DISCUSSION TITLE
Compile fails on Windows 11 Visual Studio 2019

### DISCUSSION THREAD

**Discussion by lrsweet** [OrcaSlicer/OrcaSlicer#992] [Q&A]
Compiling fails with the following error:

**fatal error C1083: Cannot open include file: 'pthread.h': No such file or directory.**

How can I resolve this issue:

Determining if the include file pthread.h exists failed with the following output:
Change Dir: C:/Users/lrswe/Downloads/OrcaSlicer-main/OrcaSlicer-main/build/CMakeFiles/CMakeTmp

Run Build Command(s):C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/MSBuild/Current/Bin/MSBuild.exe cmTC_b6e8d.vcxproj /p:Configuration=Debug /p:Platform=x64 /p:VisualStudioVersion=16.0 /v:m && Microsoft (R) Build Engine version 16.11.2+f32259642 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Microsoft (R) C/C++ Optimizing Compiler Version 19.29.30148 for x64
  Copyright (C) Microsoft Corporation.  All rights reserved.
  CheckIncludeFile.c
  cl /c /Zi /W3 /WX- /diagnostics:column /Od /Ob0 /D WIN32 /D _WINDOWS /D "CMAKE_INTDIR=\"Debug\"" /D _MBCS /Gm- /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /Fo"cmTC_b6e8d.dir\Debug\\" /Fd"cmTC_b6e8d.dir\Debug\vc142.pdb" /external:W3 /Gd /TC /errorReport:queue "C:\Users\lrswe\Downloads\OrcaSlicer-main\OrcaSlicer-main\build\CMakeFiles\CMakeTmp\CheckIncludeFile.c"
C:\Users\lrswe\Downloads\OrcaSlicer-main\OrcaSlicer-main\build\CMakeFiles\CMakeTmp\CheckIncludeFile.c(1,10): fatal error C1083: Cannot open include file: 'pthread.h': No such file or directory [C:\Users\lrswe\Downloads\OrcaSlicer-main\OrcaSlicer-main\build\CMakeFiles\CMakeTmp\cmTC_b6e8d.vcxproj]

**There is also a problem with OpenCASCADE missing .cmake file.  See errors below.**

  CMake Error at src/libslic3r/CMakeLists.txt:447 (find_package):
  By not providing "FindOpenCASCADE.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "OpenCASCADE", but CMake did not find one.

  Could not find a package configuration file provided by "OpenCASCADE" with
  any of the following names:

    OpenCASCADEConfig.cmake
    opencascade-config.cmake

  Add the installation prefix of "OpenCASCADE" to CMAKE_PREFIX_PATH or set
  "OpenCASCADE_DIR" to a directory containing one of the above files.  If
  "OpenCASCADE" provides a separate development package or SDK, be sure it
  has been installed.

-- Configuring incomplete, errors occurred!
See also "C:/Users/lrswe/Downloads/OrcaSlicer-main/build/CMakeFiles/CMakeOutput.log".
See also "C:/Users/lrswe/Downloads/OrcaSlicer-main/build/CMakeFiles/CMakeError.log".

C:\Users\lrswe\Downloads\OrcaSlicer-main\build>cmake --build . --config Release --target ALL_BUILD -- -m
Microsoft (R) Build Engine version 16.11.2+f32259642 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1009: Project file does not exist.
Switch: ALL_BUILD.vcxproj

C:\Users\lrswe\Downloads\OrcaSlicer-main\build>cmake --build . --target install --config Release
Microsoft (R) Build Engine version 16.11.2+f32259642 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.
[CMakeError.log](https://github.com/SoftFever/OrcaSlicer/files/11416050/CMakeError.log)
[CMakeOutput.log](https://github.com/SoftFever/OrcaSlicer/files/11416051/CMakeOutput.log)

MSBUILD : error MSB1009: Project file does not exist.
Switch: install.vcxproj

**Comment by jwalker55** [comment]
I have this same problem. Did you ever fix it?
